LocalSettings.php

```diff
@@ -4600,6 +4600,18 @@
 				'edit' => true,
 			),
 		),
+		'+enigmawiki' => array(
```

Comment:
Edit is not revoked because your appending to the default and not overiding, but edit perm is controlled using wgEditNatrix